documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+# Getting Started: how to process data for Timelapse Feature Explorer
+
+The [Timelapse Feature Explorer (TFE)](https://timelapse.allencell.org) is a web-based application designed for the interactive visualization and analysis of segmented time-series microscopy data! Data needs to be processed into a specific format to be loaded into the viewer.
+
+In this tutorial, you'll learn how to prepare your data for the Timelapse Feature Explorer.
+
+## 1. Terms
+
+A few key terms:
+
+- **Dataset**: A dataset is a single time-series, and can have any number of tracked objects and features.
```

Comment:
I don't know what size of data people might bring, but it's actually limited to the 32-bit pixel size of a png for our indexing.
```suggestion
- **Dataset**: A dataset is a single time-series, and can have any number of tracked objects and features. (up to about 16.7 million unique tracked objects)
```